src/pages/Info/About/index.js

```diff
@@ -18,24 +22,160 @@ const useStyles = makeStyles(() => ({
+            About Us
+          </Typography>
+          <IconButton edge="end" className={classes.menuButton} color="inherit">
+            <MoreVertIcon />
+          </IconButton>
+        </Toolbar>
+      </AppBar>
+      <Container>
+        <ScrollToTop />
+        <div>
+          <Box>
+            <Typography variant="h4" style={{ textAlign: "center", padding: "5%" }}>
+              Code for Cause
+            </Typography>
+          </Box>
+          <br />
+          <Grid container spacing={3}>
+            <Grid item xs={8}>
+              Nulla facilisi. Phasellus sollicitudin nulla et quam mattis feugiat. Aliquam eget
+              maximus est, id dignissim quam.
+            </Grid>
+            <Grid item xs={4}>
+              <img className={classes.img} src="/assets/img/team.jpg" />
+            </Grid>
+          </Grid>
+        </div>
+        <div>
+          <Box>
+            <Typography variant="h4" style={{ textAlign: "center", padding: "5%" }}>
+              Our Vision
+            </Typography>
+            <br />
+            <Grid container spacing={3} style={{ textAlign: "center" }}>
+              <Grid item xs={12}>
+                Grow Up Your Workflow Speed.
+                <br />
+                Join our team and become a part of that believe in open education for all.
+              </Grid>
+            </Grid>
+            <br />
+            <Grid container spacing={3} style={{ textAlign: "center", color: "blue" }}>
+              <Grid item xs={4}>
+                <Typography variant="h4" style={{ textAlign: "center" }}>
+                  27,889
+                </Typography>
+              </Grid>
+              <Grid item xs={4}>
+                <Typography variant="h4" style={{ textAlign: "center" }}>
+                  90%
+                </Typography>
+              </Grid>
+              <Grid item xs={4}>
+                <Typography variant="h4" style={{ textAlign: "center" }}>
+                  70,592
+                </Typography>
+              </Grid>
+            </Grid>
+          </Box>
+        </div>
+        <br />
+        <div>
+          <Box>
+            <Typography variant="h4" style={{ textAlign: "center", padding: "5%" }}>
+              Our Team
+            </Typography>
+          </Box>
+          <br />
+          <Grid container spacing={3}>
+            <Grid item xs={4}>
+              <Avatar
+                alt="Remy Sharp"
+                src="https://www.w3schools.com/w3images/team1.jpg"
+                className={classes.large}
+              />
+              <br />
+              <Typography variant="body2" component="p" style={{ float: "left" }}>
+                Jane Doe
+              </Typography>
+              <Link href="mailto:lorem@ipsum.com?body=Hello Friends" className={classes.icon}>
+                <EmailOutlinedIcon />
+              </Link>
+            </Grid>
```

Comment:
Instead of writing the Grid item again and again make a list and and map it to return 3 Avatars.

Also show 6 avatars instead of 3, since cfc has 6 members in their team.